pkgs/development/python-modules/pyflipper/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pyserial,
+  websocket-client
+}:
+
+buildPythonPackage rec {
+  pname = "pyflipper";
+  version = "unstable-2024-04-15";
```

```suggestion
  version = "0.18-unstable-2024-04-15";
```
https://github.com/wh00hw/pyFlipper/blob/e8a82a25eb766fac53a2e6e5fff6505f60cf0897/setup.py#L11